Detectors/TRD/base/include/TRDBase/RecoParam.h

```diff
@@ -0,0 +1,57 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+/// \file RecoParam.h
+/// \brief Error parameterizations and helper functions for TRD reconstruction
+/// \author Ole Schmidt
+
+#ifndef O2_TRD_RECOPARAM_H
+#define O2_TRD_RECOPARAM_H
+
+#include <array>
+
+namespace o2
+{
+namespace trd
+{
+
+class RecoParam
+{
+ public:
+  RecoParam() = default;
+  RecoParam(const RecoParam&) = default;
+  ~RecoParam() = default;
+
+  /// Load parameterization for given magnetic field
+  void setBfield(float bz);
+
+  /// Recalculate tracklet covariance based on phi angle of related track
+  void recalcTrkltCov(const float tilt, const float snp, const float rowSize, std::array<float, 3>& cov) const;
+
+  /// Get tracklet r-phi resolution for given phi angle
+  /// \param phi angle of related track
+  /// \return sigma_y^2 of tracklet
+  float getRPhiRes(float snp) const { return (mA2 + mC2 * (snp - mB) * (snp - mB)); }
+
+  /// Get tracklet z correction coefficient for track-eta based corraction
+  float getZCorrCoeffNRC() const { return mZCorrCoefNRC; }
+
+ private:
+  // tracklet error parameterization depends on the magnetic field
+  float mA2{1.f};            ///< parameterization for tracklet position resolution
+  float mB{0.f};             ///< parameterization for tracklet position resolution
+  float mC2{0.f};            ///< parameterization for tracklet position resolution
+  float mZCorrCoefNRC{1.4f}; ///< tracklet z-position depends linearly on track dip angle
+};
```

Comment:
not sure this causes the macro test problem, but since you require a dictionary for this class in the CMakeLists and LinkDef, you should have here `ClassDefNV...`

Reply:
Thanks for spotting! I have added the `ClassDefNV` and also added RecoParam to the LinkDef (this I think I forgot before), but still no luck. I cannot load the macro locally and don't really know how to debug the problem